app/utils/file.js

```diff
@@ -215,3 +215,7 @@ function populateMaps() {
         }
     });
 }
+
+export function getVideoPathFromFile(file) {
```

Comment:
Maybe have this function accept two args, adding dir with the default on `VIDEOS_PATH ` so it can be used for `DownloadDir` as well?  Like:
```javascript
export function getVideoPathFromFile(file, dir=VIDEOS_PATH)
```

Reply:
sounds good, I'll make the changes